eclass/qt5-build.eclass

```diff
@@ -655,7 +655,8 @@ qt5_base_configure() {
 
 		# disable undocumented X11-related flags, override in qtgui
 		# (not shown in ./configure -help output)
-		-no-xkb -no-xrender
+		$([[ ${PV} != 5.8.9999 ]] && echo -no-xrender)
```

Comment:
Yeah, I was almost sure there was a better way to do it, but couldn't spot it yet.

And indeed, it will soon be needed for the other versions too, so I wasn't entirely sure how much sense it makes to do this now.